packages/embed/src/components/titles/Titles.jsx

```diff
@@ -37,7 +37,7 @@ const badgeTiers = [
 ]
 
 const getTierIcon = (balance) => {
-  const balanceWei = AUDIO(balance).value
+  const balanceWei = BigInt(balance)
```

Comment:
What is balance? Where does it come from? If it was a bigint, this would be equivalent. But if it's a number or string, then I get it... Strange to be passing around balance that way in the first place though!

Reply:
it's a string of wei audio, it would seem